src/styles/_custom.scss

```diff
@@ -304,3 +304,40 @@ div.is-rounded img {
     }
   }
 }
+
+.jf-cookies-banner {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  border-top: 1px solid $justfix-black;
+
+  img {
+    filter: invert(1) contrast(0.7);
+  }
+
+  .link {
+    display: inline;
```

Comment:
I think this also needs `cursor: pointer`

Reply:
eeek! yep